sections/futures/LeverageInput/LeverageInput.tsx

```diff
@@ -98,20 +97,20 @@ const LeverageInput: FC = () => {
 						maxValue={Number(truncateMaxLeverage)}
 						value={Number(truncateLeverage)}
 						onChange={(_, newValue) => {
-							onLeverageChange(newValue as number);
+							onLeverageChange(newValue.toString());
 						}}
 					/>
 				</SliderRow>
 			) : (
 				<LeverageInputContainer>
 					<StyledInput
 						data-testid="leverage-input"
-						value={leverageInput}
+						value={leverageInput === '0' ? '' : leverageInput}
```

Comment:
The current state is this:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/10401554/210827023-d90a2058-4513-4d1d-81a3-d55df84a9db9.png">

The change here means you can't type 0, but also prevents the input from displaying 0 as if it's an input when the inputs should be blank.

Reply:
but this means I can't type 0.5 right?

Reply:
We may need an additional check to see if there is a dot in the input before removing `0`. 
This is a rare case as not too many users will set a leverage size of less than 1.

Reply:
Yeah exactly, if we change this line back we will display this `0` input when the inputs are cleared and the user will be able to input `0.5`

Reply:
I will just fix both on this PR. It looks like the difference is using a local `useState` value for the display instead of displaying the value from the selector.

Reply:
if the selector / state item is never formatted then we could prob just have the raw string end to end and keep as is, only format it when it's used in an action, might a better option than mixing local and global states.

Reply:
But doesn't this miss the case where you edit order size and the leverage input is updated? I had a commit in progress that I think does it by mixing local/global state

Reply:
I had a play as wasn't sure if what I had in my mind made sense but I think it works ok, opened a PR here #1837 but let me know if I'm missing something?